examples/vello_editor/src/main.rs

```diff
@@ -137,56 +141,61 @@ impl ApplicationHandler for SimpleVelloApp<'_> {
                         GenericFamily::SystemUi.into(),
                     ])),
                 ]);
+                render_state.window.request_redraw();
```

Comment:
Does moving this here change anything or just makes it clear it happens after any state update?